src/LinuxTracing/LibunwindstackMultipleOfflineAndProcessMemory.cpp

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "LibunwindstackMultipleOfflineAndProcessMemory.h"
+
+#include <absl/base/casts.h>
+
+namespace orbit_linux_tracing {
+size_t LibunwindstackMultipleOfflineAndProcessMemory::Read(uint64_t addr, void* dst, size_t size) {
+  const uint64_t addr_start = addr;
+  const uint64_t addr_end = addr + size;
+
+  bool found_overlap = false;
+
+  // If the requested address range is entirely in the stack sample's address range, read from the
+  // stack buffer.
+  for (LibunwindstackOfflineMemory& stack_slice : stack_memory_slices_) {
```

Comment:
The variable is a `Memory`, not a `StackSlice`(`View`), so I think this sould be called `stack_memory`.